test/Confluent.Kafka.IntegrationTests/Tests/DeserializingConsumer_Poll_Error.cs

```diff
@@ -43,8 +43,8 @@ public static void DeserializingConsumer_Poll_Error(string bootstrapServers, str
             using (var producer = new Producer(producerConfig))
             {
                 firstProduced = producer.ProduceAsync(topic, Encoding.UTF8.GetBytes("key"), null).Result.TopicPartitionOffset;
-                producer.ProduceAsync(topic, null, Encoding.UTF8.GetBytes("val"));
```

Comment:
`secondProduced` isn't used and furthermore it has a different type to `firstProduced`, so better not to make the change on this line IMO.